scripts/vellum-to-hq

```diff
@@ -107,7 +107,10 @@ def build_vellum(path, branch, test=True):
 
 
 def get_old_vellum_rev(git):
-    commit = str(git('log', '--oneline', '--grep', 'vellum:', '-n', '1'))
+    commit = str(git('log', '--oneline', '--perl-regexp', '--grep', 'vellum:', '-n', '1'))
+    if 'Revert' in commit:
+        commit = str(git('log', '--oneline', '--perl-regexp', '--grep', '(?<!Revert ")vellum:', '-n', '2'))
+        commit = commit.rstrip().split('\n')[-1]
```

Comment:
Not sure if we care, but this will not work if there were multiple reverts in a row. Maybe it would be better to make the code that lists changes check if the most recent vellum commit contains "Revert" and simply display a message that says it couldn't find the latest version of Vellum.

Alternately, you could get the version that's currently live on HQ:
```py
>>> import requests
>>> resp = requests.get("https://www.commcarehq.org/static/app_manager/js/vellum/version.txt")
>>> resp.text
u'1cc6efea090874057cb4a3e8a29f4eebcc403626\n'
```